static/sass/_pattern_inline-images.scss

```diff
@@ -0,0 +1,56 @@
+@mixin jp-p-inline-images {
+    @include vf-p-inline-images;
+  
+    .p-inline-images {
+      &__item--smaller {
+        margin: $sp-medium;
+        max-width: 7.25rem;
+      }
+  
+      &--smaller {
+        &__item {
+          margin: 1rem;
+          max-height: inherit;
+          max-width: inherit;
+  
+          @media only screen and (min-width: $breakpoint-medium) {
+            margin: 1.875rem;
+          }
+        }
+  
+        img {
+          max-height: map-get($icon-sizes, thumb--small);
+          max-width: 5rem;
+          width: auto;
+  
+          @media screen and (min-width: $breakpoint-medium) {
+            max-height: 3.5rem;
+            max-width: 7rem;
+          }
+        }
+      }
+    }
+  
+    // XXX Peter Mahnke 2018-03-01
```

Comment:
I think that this comment can be removed. p-inline-images is being deprecated upstream in Vanilla.

Reply:
I ended up adding in the snapcraft solution (mostly).